README.md

```diff
@@ -74,39 +44,177 @@ This program was created with the following tech stack and packages:
+    4. Delete the `extras` folder which is currently located at `speaking-portal-project\rhubarb\extras`
+5. In the terminal, navigate to `speaking-portal-project`
+6. Run `npm install` to ensure all packages are installed
+7. Run `npm start` to start the API
+8. Check in the terminal that the API is listening
+9. Create the user's Kukarella files:
     1. Go to [Kukarella](https://www.kukarella.com/)
-    2. Enter text, select a language, select a voice, and convert to audio
+    2. Enter text, select a language, select a voice, and convert the text to audio
     3. Download the audio as a .wav file
     4. Download the text file
-4. Move the Kukarella files to `speaking-portal-project\rhubarb`
-5. In the terminal, navigate to `speaking-portal-project`
-6. Install ffmpeg
-   1. Follow the steps on [adamtheautomator.com](https://adamtheautomator.com/install-ffmpeg/) for your corresponding
-   operating system. We recommend the Powershell approach for Windows users.
-7. Run `npm install`
-8. Run `npm start`
+10. Use Postman or a similar platform to send a POST request (see [POST Request Properties](#post-request-properties))
+to the local API that is currently listening for a request from step 7.
+11. Wait for the API to process the request and return the video
+12. Enjoy the animation!
 
 ## How it Works
 
-The Kurkarella TTS app creates the synthesized speech file from the user’s voice and text input. The Animation Add-On
-is broken down into two main components: the phoneme factory and the animation processor. The phoneme factory is in
-charge of mapping spoken language from an audio file into a series of phonemes, which are units of sound that
-distinguish one word from another. In order to make this process as accurate as possible, the phoneme factory also
-receives the user text file and language selection.
+### Overview
+
+The Speaking Portal Project (SPP) is built to connect with Kukurella's Text-to-Speech platform as an API. The SPP is
+broken down into three main components: **The API**, **The Phoneme Factory**, and **The Animation Factory**.
+
+The Speaking Portal API generates an MP4 animation from the speech file and text file received from Kukarella's
+TTS process. Text and audio inputs are first processed and converted into a json file containing phonemes and their
+timings. This processing is done via the [Rhubarb library](https://github.com/DanielSWolf/rhubarb-lip-sync). (Phonemes
+are the smallest unit of sounds that uniquely identify distinct sounds within words.) Once a phoneme file is created,
+the animation process then identifies which avatar assets to use based on the phoneme. Other features, such as blinks
+and pose changes are added, and then all image assets are rendered in a video file through ffmepg.
+
+### API
+
+Once the API receives the request, it initializes a node instance, creates a `/tmp` directory for file I/O operations,
+and begins the animation process by sending all user inputs to main.
```

Comment:
```suggestion
When the API receives a request, the uploaded files are stored in the `/tmp` directory. The api code then reads these inputs and converts them to the appropriate format for Rhubarb and ffmpeg. Upon conversion, the animation process is started by sending all user inputs to the `main.ts` file. 
```

Reply:
Technical what is described originally is what happens when the API is started for the first time, not upon request

Reply:
if you can edit api to API, I'll commit the suggestion!!